lib/bracket-matcher.coffee

```diff
@@ -24,6 +24,7 @@ class BracketMatcher
     "‹": "›"
 
   toggleQuotes: (includeSmartQuotes) ->
+    return if @pairedCharacters? and includeSmartQuotes is (@pairedCharacters isnt @defaultPairs)
```

Comment:
Got it. It seems, however, that this method gets called only on `insertText` and `backspace`: what do you think about removing it if there's no serious slowdown?